src/plugins/intel_npu/src/backend/include/zero_utils.hpp

```diff
@@ -16,19 +16,31 @@ namespace intel_npu {
 
 namespace zeroUtils {
```

Comment:
Could you consider moving this file to the utils folder?

Reply:
Of course, thank you for the idea